src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
-body {
+$cell-size: 100px;
+$default-month-length: 31;
+$days-of-week: (
+  'mon': '0',
+  'tue': '1',
+  'wed': '2',
+  'thu': '3',
+  'fri': '4',
+  'sat': '5',
+  'sun': '6',
+);
+$gap: 1px;
+
+* {
   margin: 0;
+  padding: 0;
+}
```

Comment:
The universal selector (*) is known to be slow and should be avoided if possible. Consider using more specific selectors to apply the reset styles.